client/src/assets/images/accessCycleHeader6.svg

```diff
@@ -0,0 +1,9 @@
+<path d="M105 67C105 61.4772 109.477 57 115 57H205C210.523 57 215 61.4772 215 67V134C215 139.523 210.523 144 205 144H115C109.477 144 105 139.523 105 134V67Z" fill="#050505"/>
+<path d="M129.35 120.8L145.15 105L129.35 89.2L126.083 92.4L138.683 105L126.083 117.6L129.35 120.8ZM164.5 96.5333C164.7 96.1333 165.033 95.7333 165.3 95.4C167.1 93.1333 169.833 92.1333 172.9 92.1333C175.367 92.0667 177.833 92.6667 179.7 94.0667C181.3 95.2 182.3 97 182.567 98.9333L175.633 99.9333C170.167 100.733 166.5 101 162.633 103.2C159.233 105.2 157.167 108.6 157.167 112.467C157.167 115.133 157.967 117.067 159.5 119.333C161.433 121.933 165.1 123.8 168.767 123.933C171.833 124.067 174.9 123.333 177.7 121.8C179.967 120.667 182.367 118.533 183.5 116.133V123H189.233V100.4C189.233 98.3333 188.767 96 187.7 93.8C185.033 88.4 179.633 86.0667 173.033 86.0667C167.433 86.0667 162.7 88 159.5 92.2667L158.567 93.6667L164.5 96.5333ZM182.833 104.333C182.833 106.6 182.833 108.667 181.633 111.133C179.833 115.733 174.233 118.2 169.7 118C166.167 117.8 163.567 115.067 163.633 112.067C163.7 107.467 169.367 106.533 174.7 105.667C178.433 105 180.767 104.667 182.833 104.333Z" fill="#FED136"/>
```

Comment:
Good job with the new svg files! It looks like there's a lot of logo and other image files in the assets folder now (including the old logo), so I think it might be a good idea to remove those from the repo and put them somewhere else for historical documentation? Just since we don't reference them in the main site anymore